.github/workflows/brs-commit-checker.yml

```diff
@@ -0,0 +1,64 @@
+name: BRS Commit Checker Report
+on:
+      workflow_dispatch:
+        inputs:
+            fix_version:
+                type: string
+                required: true
+                description: The ICU Jira "Fix Version" semver
+            from_git_ref:
+                type: string
+                required: true
+                description: The git ref start of comparison range. Prefix branches with `origin/`.
+            end_git_ref:
+                type: string
+                required: true
+                description: The git ref end of comparison range. Must be descendant of `from_git_ref`. Prefix branches with `origin/`.
+            # Jira user name & API token is used for processing sensitive tickets comes from Github Secrets
+            # stored in the repository
+
+jobs:
+    commit-report:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                fetch-tags: true
+                fetch-depth: 0
+            # workaround for bug in checkout action. this step should be redundant.
```

Comment:
If it's a problem in the checkout action does it mean we have to do it in all of our actions?

Reply:
No, because most actions only care about the latest version of code that was created as a result of the PR's latest commit. The default behavior of `actions/checkout` is to give you that. 

In contrast, this workflow actually cares about being able to see all branches and tags of the repository, which is why we have to deal with this workaround here (and not yet anywhere else).